sauron/sauron.py

```diff
@@ -21,6 +23,17 @@ def fetch(url):
     # transaction a certain amount of times.
     session = requests.session()
     session.proxies = plugin.sauron_socks_proxies
+    retry_strategy = Retry(
+        backoff_factor=1,
+        total=10,
+        status_forcelist=[429, 500, 502, 503, 504],
+        method_whitelist=["HEAD", "GET", "OPTIONS"]
+    )
+    adapter = HTTPAdapter(max_retries=retry_strategy)
+
+    session.mount("https://", adapter)
+    session.mount("https://", adapter)
```

Comment:
This line is duplicated, maybe you wanted both http and https ?

Reply:
Correct, that's what I wanted to do :-)